app/src/main/res/layout/activity_constraint.xml

```diff
@@ -0,0 +1,119 @@
+        android:layout_marginTop="@dimen/spacing24"
+        android:id="@+id/button_clasified"
+        android:text="@string/button_classified"
+        style="@style/ShowreelLabel.White.ButtonTextSmall"
+        android:background="@drawable/button_small_type1"
+        android:drawableLeft="@drawable/ic_label"
+        app:layout_constraintTop_toBottomOf="@id/header"
+        app:layout_constraintStart_toEndOf="@id/button_motor"
+        />
+
+    <android.support.v7.widget.AppCompatButton
+        android:layout_width="@dimen/spacing110"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="@dimen/spacing16"
+        android:layout_marginTop="@dimen/spacing8"
+        android:id="@+id/button_services"
+        android:text="@string/button_services"
+        android:background="@drawable/button_small_type2"
+        android:drawableLeft="@drawable/ic_label"
+        style="@style/ShowreelLabel.White.ButtonTextSmall"
+        app:layout_constraintTop_toBottomOf="@id/button_clasified"
+        app:layout_constraintStart_toEndOf="@id/button_motor"
+        />
+
+    <android.support.v7.widget.AppCompatButton
+        android:layout_width="@dimen/spacing110"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/spacing24"
+        android:layout_marginLeft="@dimen/spacing16"
+        android:id="@+id/button_properties"
+        android:text="@string/button_properties"
+        android:background="@drawable/button_small_type3"
+        android:drawableLeft="@drawable/ic_label"
+        style="@style/ShowreelLabel.White.ButtonTextSmall"
+        app:layout_constraintTop_toBottomOf="@id/header"
+        app:layout_constraintStart_toEndOf="@id/button_clasified"
+        />
+
+    <android.support.v7.widget.AppCompatButton
+        android:layout_width="@dimen/spacing110"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="@dimen/spacing16"
+        android:layout_marginTop="@dimen/spacing8"
+        android:id="@+id/button_jobs"
+        android:text="@string/button_jobs"
+        android:background="@drawable/button_small_type4"
+        android:drawableLeft="@drawable/ic_label"
+        style="@style/ShowreelLabel.White.ButtonTextSmall"
+        app:layout_constraintTop_toBottomOf="@id/button_properties"
+        app:layout_constraintStart_toEndOf="@id/button_services"
+        />
+
+    <android.support.v7.widget.RecyclerView
+        android:layout_width="@dimen/wrap_constraint"
+        android:layout_height="@dimen/wrap_constraint"
+        android:scrollbars="vertical"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/button_motor"
+        app:layout_constraintBottom_toTopOf="@id/bottom_navigation"
+        tools:listitem="@layout/item_constraint"
+        tools:itemCount="3">
+
+    </android.support.v7.widget.RecyclerView>
```

Comment:
Poszukaj też jeszcze takich rzeczy jak tutaj ze otwierasz recyclerView tylko po to zeby linijke pozniej go zamknąć i je popraw zeby byly jak w buttonie wyzej